projects/components/facet/bootstrap/facet-multi/facet-multi.component.ts

```diff
@@ -6,7 +6,7 @@ import { FacetService } from '../../facet.service';
 
 export interface FacetConfig {
   name: string;
-  type: 'list' | 'tree';
+  type: 'list' | 'tree' | 'tagcloud';
```

Comment:
Since we don't have a tag cloud component at the moment, can you uncommit this for now? We'll add it when the new component is available.

Reply:
Sure, I'll backtrack to the previous commit and make my change without the tagcloud.